docs/make.jl

```diff
@@ -26,7 +26,7 @@ makedocs(
             "Categorical Data" => "man/categorical.md",
             "Missing Data" => "man/missing.md",
             "Data manipulation frameworks" => "man/querying_frameworks.md",
-            "Comparison with Stata/R" => "man/comparisons.md"
+            "Comparison with Python/R/Stata" => "man/comparisons.md"
```

Comment:
Which is better, "Python/R/Stata", "pandas/dplyr/Stata", or just "other languages"? Note that we'll probably add data.table at some point.

Reply:
Not trying to put Stata under but most people work with Python and R these days. Maybe "Comparison with Python, R, etc."?

Reply:
I think it is OK to keep it as is for now. We can change it later if we add other options

Reply:
OK